### PR TITLE
os/board/rtl8721csm: support WPA3 128 bytes passphrase

### PR DESCRIPTION
**Pull request by gexuyan**

### CODE REVIEW

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Comment by j11591** [comment]
Looks good to me

**Review by pillip8282 — COMMENTED** [review]
How did you verify this feature. Now wifi manager defined passphrase length 64(WIFIMGR_PASSPHRASE_LEN, wifi_manager/wifi_manager.h). So it can't pass 128 bytes to a driver. Please share how did you verify it.

File: os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c [hunk 1]
```diff
@@ -502,7 +502,7 @@ void restore_wifi_info_to_flash(void)
 		}
 
 		memcpy(wifi_data_to_flash.psk_essid, psk_essid[index], sizeof(wifi_data_to_flash.psk_essid));
-		if (strlen((char const*)psk_passphrase64) == 64) {
+		if (strlen((char const*)psk_passphrase64) == IW_WPA2_PASSPHRASE_MAX_SIZE) {
```
Comment (jeongarmy) [inline os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c:505]: I think you need to add `#include ieee80211.h` for `IW_WPA2_PASSPHRASE_MAX_SIZE`.

**Comment by gexuyan** [comment]
> How did you verify this feature. Now wifi manager defined passphrase length 64(WIFIMGR_PASSPHRASE_LEN, wifi_manager/wifi_manager.h). So it can't pass 128 bytes to a driver. Please share how did you verify it.

Sorry this commit may have some problem with the latest master branch, I will close it first and raise a new one asap. Thank you!

**Pull request closed without merging**